db/migrate/20150828153331_change_business_support_edition_areas_to_gss_codes.rb

```diff
@@ -0,0 +1,20 @@
+class ChangeBusinessSupportEditionAreasToGssCodes < Mongoid::Migration
+  def self.up
+    # Editing of an edition with an Archived artefact is not allowed
+    Edition.skip_callback(:save, :before, :check_for_archived_artefact)
+
+    BusinessSupportEdition.all.each do |edition|
+      edition.area_gss_codes = Area.areas_for_edition(edition).map(&:gss).compact
```

Comment:
Since we decided to [nest the GSS codes](https://github.com/alphagov/imminence/blob/master/app/presenters/areas_presenter.rb#L39-L41) we actually need `codes["gss"]` here.